src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleThreadContext.java

```diff
@@ -54,6 +55,11 @@ public class ModuleThreadContext extends StarlarkThreadContext {
   private final Map<String, ModuleOverride> overrides = new HashMap<>();
   private final Map<String, RepoNameUsage> repoNameUsages = new HashMap<>();
 
+  private final Map<String, ModuleExtensionUsageBuilder.RepoOverride> reposSubjectToRepoOverride =
```

Comment:
these two can similarly (see other comment) be renamed to just `overriddenRepos` and `overridingRepos`.